opm/common/OpmLog/LogBackend.hpp

```diff
@@ -75,6 +79,7 @@ namespace Opm
         int64_t m_mask;
         std::shared_ptr<MessageFormatterInterface> m_formatter;
         std::shared_ptr<MessageLimiter> m_limiter;
+        int64_t m_level;
```

Comment:
As I see it there are two different axes for configuring the logging:

1. The type of message; i.e. `Info`, `Warning`,  `Error`, `F... up` and so on. These types are already "supported"; since we wish to configure ccombinations like `F...up | Error` they are implemented as bitmasks.

2. For a given message type - how much should be logged; i.e. a form of verbosity. This is not supported in the current implementation.

Of course the differentiation between messagetype and verbosity can be less than clear - maybe we should not even think of supporting it. The original choice of using `int64_t` for the messagetype was to leave enough room in the bitmask that we could possibly at a later stage (now?) combine messagetype and verbosity in one combined mask - maybe over engineered?